src/modules/Exposure/info.json

```diff
@@ -6,9 +6,9 @@
           "type": "range",
           "desc": "exposure value for the new image",
           "default": "0",
-          "min": "-4",
+          "min": "-3",
```

Comment:
Why -3 as min? Actually providing even -10 and +10 will not cause any problem since only the min is chosen. Will it use more cpu power though??

Reply:
By min I meant the Math.min

Reply:
Actually I think 8 and -8 will be the best since 2^8=256.

Reply:
Try it out on your machine once. Every negative value after -3 provides the very same output, so there is no point for all those values. I put these after refining the logic, the step size is small enough to detect subtle changes and the range is short enough to make every stop distinct

Reply:
If the values of r,g,b were higher then maybe it would provide a different value? I'm using my phone btw. Try using exposure=4 first and then run the exposure module again on the same image

Reply:
Maybe its not doing that since the division might not be an integer. Try using Math.floor

Reply:
@HarshKhandeparkar I don't think that changing the values will achieve much. A -3 output was equivalent to a -4 output on the scale, so I used a -3. Similarly on the other end. That's the reason I've kept these limits, anything beyond this range is almost identical to the extremes, so it wouldn't make much sense to provide a scale that doesn't produce a different output.
@jywarren thoughts?

Reply:
I think it's fine to have limits like this. If people want to push this later, we can make more changes!